docs/usage/frameworks/litestar.rst

```diff
@@ -397,12 +396,11 @@ If you use a different path than `./migrations`, be sure to also set this in you
         ),
     )
 
-And then run the following:
+And then run the following to initialize the migrations directory:
 
-.. code-block:: bash
+.. code-block:: shell-session
 
-    # Initialize migrations directory
-    litestar database init ./alembic
+    $ litestar database init ./alembic
```

Comment:
note: without the `$` pygments doesn't highlight correctly, and the text becomes too low contrast